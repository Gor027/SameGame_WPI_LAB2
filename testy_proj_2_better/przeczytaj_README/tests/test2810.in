 WWWC CC  C   CWW  CWWWCC
 WWW  CCWC WCWCCWCCWCCWCW W W WW WCWW   CWWC 
WC W C WWW WCWWW WCCCWCWWCCCC  WCC  W CW  C W WW CC  CWC CCCCWCC C W    WC  W C
 CC  C  W 
WWC WWW  CW  WCWC CW WC WWWC C WWCCCCW C C CWC      CCCWWWWC C WC WWCCW C
C  WWWWWCWWW   CC WCC WW  WC WC W CCW C C   W C CWWWWCWWC W WWWCW C CWWW CC
CWCWCWWW  W   CWW   C WWC  CCCWCWWWW   W WC WWWWCCWW C   W  WWC  CW C C WC
