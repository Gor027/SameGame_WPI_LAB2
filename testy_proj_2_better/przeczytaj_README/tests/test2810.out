 W             W      W
 WWW CCC      CCW  CW WCC
 CWWCCCCWCC   CWW CWCWCCW WC   WCWCWW       W        C       W
 CCWWWWWWWWW WWWC CCCCCWW CCWCWWWCCCW  W WW W W  CC  CWC CC CCCC  W C W WC
WWCWWWWWWCWWCWCCCCCWCWWWWCWCCWCWWCCCC CCCCCCWCCWCWWWCCCWWWWCWWWWCCCWCCW CC
WWCWCWWWCWWWCWCWWWWCCWWWCWWCCCWCWWWWWWCWCWCCWWWWCCWWWCWWCWWCWWCCWCWWCWCWWCCWC
